Buttons/Buttons.css

```diff
@@ -0,0 +1,160 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  --theme-color: #490e60;
+  --theme-text-color: #f7f7f7;
+  --theme-hover-color: #6d3e7f;
+  --primary-color: #e6e6fa;
+  --primary-text-color: #490e60;
+  --primary-hover-color: #cfcfe1;
+  --secondary-color: #326771;
+  --secondary-text-color: #071203;
+  --secondary-hover-color: #28525a;
+  --success-color: #84f1c9;
+  --success-text-color: #1b7052;
+  --danger-color: #fcc5c5;
+  --danger-text-color: #dc3545;
+  --warning-color: #ffe7c5;
+  --warning-text-color: #ff9204;
+}
+
```

Comment:
You can move all the variables to `variables.css`. So that you don't have to repeat every file. Also, you can put variables inside `:root{}`